crytic/slither
# Merge 0.9.1 fix

### Notes

#### Rev1
This fixes some issues that were missed initially when merging in the 0.9.1 source code. We have changed all occurrences of `contract` to `contract_kind`, and also passed source mapping to output code in their json form.

### Testing
* Check out the `merge-0.9.1` branch of `slither-task`, run `make update` and `pipenv install`, and then replace the `utilities/slither` directory with this commit.
* Run `./evaluate.sh run 100` then `./evaluate diff`. Only the `nfa` project fails where it was previously recorded as successful. The `nfa` project has missing source code, so it shouldn't succeed.

### Related Issue
https://github.com/orgs/CertiKProject/projects/6/views/1?filterQuery=-no%3Astatus+kevinclancy
